proc_scripts/plotting.py

```diff
@@ -124,3 +124,52 @@ def plot_curve(fl, f, name, guess=None):
             fl.plot(guess, '-', label='initial guess')
         text(0.75, 0.25, f.latex(), transform=gca().transAxes, size='large',
                 horizontalalignment='center',color='k')
+    def next(self, *arg, **kwargs):
+        kwargs.update({"figsize": (9, 6), "legend": True})
+        super().next(*arg, **kwargs)
+
+    def complex_plot(fl, d, label="", show_phase=False, show_real=True):
```

Comment:
needs a docstring

Reply:
You need to give the parameters and the return value in the docstring, always

Reply:
done